### ISSUE TITLE
BUG: links for "calculate your exact net price" take you to the FA page, not the calc

### ISSUE THREAD

**Issue by m-hsiang** [18F/college-choice#621]
clicking on the link takes you to the MIT financial aid page, but not to the calculator, which is still relatively challenging to find

<img width="891" alt="screen shot 2015-08-26 at 2 40 38 pm" src="https://cloud.githubusercontent.com/assets/1036641/9502664/b0fd9f42-4c00-11e5-91b6-76d342265935.png">

<img width="1154" alt="screen shot 2015-08-26 at 2 42 50 pm" src="https://cloud.githubusercontent.com/assets/1036641/9502685/c1d1802c-4c00-11e5-94bb-4e36c7f9a990.png">

**Comment by ErieMeyer** [comment]
Super interesting. I wonder if there's a way to share this feedback to the schools. Will ping Clare.

**Comment by ErieMeyer** [comment]
There is! Collect any where you see this behavior and we'll report it back to IPEDS.

**Comment by m-hsiang** [comment]
Listing schools that I've checked:
MIT
St Paul School of nursing

**Comment by saintsoup52** [comment]
This is probably an issue in the data itself.

These URLs are reported to ED by the schools and a number of groups have done audits of these links only to conclude that there is not solid compliance to the standards ED has set out for this data.

**Comment by m-hsiang** [comment]
Yup, but I was told by erie to keep track of them, as there is a pathway to circle back with the schools on it. 

----
Mina Hsiang
c: (617) 866 9150

My apologies for any phone-related typos.

**Comment by LisaGee** [comment]
mail sent to Clare 8/28/15

**Comment by LisaGee** [comment]
@claremcann - Any next steps on this?
